code/datums/emergency_calls/emergency_call.dm

```diff
@@ -28,6 +28,8 @@
 	var/mob_min = 3
 	var/dispatch_message = "An encrypted signal has been received from a nearby vessel. Stand by." //Msg to display when starting
 	var/arrival_message = "" //Msg to display about when the shuttle arrives
+	var/chance_hidden = 20
```

Comment:
bit of a nitpick, but maybe this should be a define in the file instead since the variable doesn't change. I get that this entire file does variables instead of defines this way but idk.